docs/source/how_to_guides/configure_cloud_storage_cred.md

```diff
@@ -0,0 +1,101 @@
+# Configure Cloud Storage Credentials
+
+Streaming dataset supports the following cloud storage providers to stream your data directly to your instance.
+- [Amazon S3](https://aws.amazon.com/s3/)
+- [Google Cloud Storage](https://cloud.google.com/storage)
+- [Oracle Cloud Storage](https://www.oracle.com/cloud/storage/)
+
+## Amazon S3
+
+For S3 bucket with a public access, no additional setup is required, simply specify the S3 URI of the resource.
```

Comment:
```suggestion
For an S3 bucket with public access, no additional setup is required, simply specify the S3 URI of the resource.
```

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/169/commits/c164c1329e2a1cdfc5d49ece7647e05bca8ce411.